package.json

```diff
@@ -24,10 +25,7 @@
   "jest": {
     "verbose": false,
     "testRegex": ".*/*.spec.ts$",
-    "moduleFileExtensions": [
-      "js",
-      "ts"
-    ],
+    "moduleFileExtensions": ["js", "ts"],
```

Comment:
Looks like this is nothing but formatting, can you revert this change?  I need to add `package.json` to `.prettierignore`.